ctx.go

```diff
@@ -736,6 +731,18 @@ func (c *Ctx) JSONP(data interface{}, callback ...string) error {
 	return c.SendString(result)
 }
 
+// XML converts any interface or string to XML.
+// This method also sets the content header to application/xml.
+func (c *Ctx) XML(data interface{}) error {
```

Comment:
use `any` considering we are already using go1.18?

Reply:
This is v2

Reply:
oh sorry 😥

Reply:
No problem. I'll convert it to any when merging master into the v3-beta.